admin/config.js

```diff
@@ -99,5 +99,10 @@ module.exports = {
   },
   Cors: {
     Whitelist: process.env.CORS_WHITELIST || 'http://localhost:4200' // for development
+  },
+  Tokens: {
+    // 12 hours default expiry
+    jwtTimeOutHours: process.env.JWT_TIMEOUT_HOURS || 12,
+    sasTimeOutHours: process.env.SAS_TIMEOUT_HOURS || 12,
```

Comment:
I personally like, and agree with, trailing commas, but this is raising a linting error. Should we update the linting rules? I do agree with this line.

Reply:
GDS pretty specify that standard is the linter of choice.   `standard` is not configurable, so I will update this trailing comma in another branch.